deploy/ovms/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ovms
+  labels:
+    app: ovms
+spec:
+  selector:
+    matchLabels:
+       app: ovms
+  replicas: {{ .Values.replicas }}
+  template:
+    metadata:
+      labels:
+        app: ovms
+    spec:
+      containers:
+      - name: ovms-resnet
+        image: intelaipg/openvino-model-server:latest
+        ports:
+        - containerPort: 8080
+        - containerPort: 8081
+        readinessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        env:
+        - name: "GOOGLE_APPLICATION_CREDENTIALS"
+          value: /secret/gcp-creds.json
+        command: ["/ie-serving-py/start_server.sh"]
+        args: ["ie_serving", "model",
+               "--model_path", "gs://models-repository",
+               "--model_name", "icnet-camvid-ava-0001",
+               "--port", "8080",
```

Comment:
I propose to add the model name as a parameter in the "values.yaml" file - as right now this deployment can serve this one model only. Please add also an information how to pass this name while starting the deployment - in a similar manner to a description of passing a number of replicas.

Reply:
Yes, please make sure we provide instructions for multiple models.

Reply:
Done. I parametrized many things in _values.yaml_ and provided instruction for multiple models in _Deploy the Model Server with configuration file_ section.